src/librustc/session/config.rs

```diff
@@ -109,6 +109,7 @@ impl OutputType {
             OutputType::Bitcode => "bc",
             OutputType::Assembly => "s",
             OutputType::LlvmAssembly => "ll",
+            OutputType::Object if cfg!(target_os = "windows") => "obj",
             OutputType::Object => "o",
```

Comment:
I'd liked to pass `Session` and use the `sess.target.target.options.target_family`, but `Session` is not `Sync`, and It can be sent in `run_work_multithreaded`.
I wonder how should I pass that information (because it will has to be sent to many functions, so I wanted it to be part of something like `WorkItem` or `CodegenContext`).